packages/lib/gpcircuits/test/bounds.spec.ts

```diff
@@ -6,6 +7,9 @@ import {
 } from "../src";
 import { circomkit } from "./common";
 
+// Order of the BN254 scalar field.
+const p = BABY_JUB_NEGATIVE_ONE + 1n;
```

Comment:
Interesting.  I forgot that we don't already have a constant for P only P-1.  I'd suggest adding BABY_JUBJUB_PRIME.
You could actually import this from ffjavascript, though that has a setup cost, so the hard-coded constant seems to be the way to do it (it shows up in test data in several other libraries in our repo).